src/main/scala/fi/oph/kouta/repository/sessionDAO.scala

```diff
@@ -40,7 +40,7 @@ object SessionDAO extends SessionDAO with SessionSQL {
     runBlockingTransactionally(deleteSession(ticket), timeout = Duration(10, TimeUnit.SECONDS)).get
 
   override def get(id: UUID): Option[Session] = {
-    runBlockingTransactionally(getSession(id), timeout = Duration(2, TimeUnit.SECONDS)).get.map {
+    runBlocking(getSession(id), timeout = Duration(2, TimeUnit.SECONDS)).map {
```

Comment:
Nyt voi nähdäkseni käydä näin:
- Request 1 kutsuu getSessionQuery ja saa session
- Sessio vanhenee
- Request 2 kutsuu getSessionQuery ja saa nonen
- Request 2 deletoi session kannasta
- Request 1 yrittää virkistää session. Se ei oikeasti onnistu, koska sessio on deletoitu. Silti se palauttaa session kutsujalle ikään kuin se olisi vielä olemassa

Tämän esitämiseksi tässä 

https://github.com/Opetushallitus/kouta-backend/blob/2b17b6be87ac9e21cd56bf217029456882c9c95e/src/main/scala/fi/oph/kouta/repository/sessionDAO.scala#L75

pitää vielä varmistua siitä, että oikeasti siellä oli yhä se sessio kannassa. Jos ei ole, palautetaan None. Ei riitä, että katsotaan, että update palauttaa jotain muuta kuin 0, sillä se voi palauttaa 0, jos sessio on kannassa mutta sitä on jo päivitetty tässä välissä rinnakkaisesti. Eli pitänee laitaa esim. returning siihen updateen ja palauttaa kannasta uudelleen vaikka cas ticket. Jos sitä ei ole, tiedetään, että sessio onkin deletoitu tässä välissä ja voidaan palauttaa None.